### ISSUE TITLE
#Include <Name> incorrectly marked as invalid

### ISSUE THREAD

**Issue by Lexikos** [thqby/vscode-autohotkey2-lsp#39]
`#include <D>` is incorrectly marked as an error ("Invalid file path") when `Lib\D.ahk` exists under the same directory as AutoHotkey.exe (i.e. D.ahk is in the "standard library"). The path to AutoHotkey.exe is set in the workspace launch.json (and is correctly used by the run/debug commands).

**Comment by Lexikos** [comment]
I realized that the extension has its own setting for "Interpreter Path", so maybe this is to be expected.

I eventually worked out that this incorrect error mark only appeared because an update to the extension renamed the setting `AutoHotkey2.Path` to `AutoHotkey2.InterpreterPath`, causing my previously configured path to be ignored. Although I admit to not checking the release notes beforehand, now that I do check them I see no mention of this change.
